cypress/integration/login.js

```diff
@@ -21,7 +21,8 @@ describe('Sourcebooks login', function() {
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('Demo User');
+        cy.get('.user-info__title').contains('Nadežda Žarikova');
         cy.get('.main-nav').find('li').should('have.length', 1);
+        cy.get('.calendar__date').contains(new Date().getDate());

```

Comment:
Great function! However, don't user "calendar__date", because this class can be found in every single day's cell. Try using "calendar--today" instead :)